packages/drivers/src/servers.ts

```diff
@@ -0,0 +1,66 @@
+import * as ds from "@devicescript/core"
+import { SensorServer, startServer } from "@devicescript/server"
+
+export interface TemperatureOptions {
+    min: number
+    max: number
+    error: number
+    read: () => Promise<number>
+}
+
+export class TemperatureServer
+    extends SensorServer<ds.ITemperatureServer>
+    implements ds.ITemperatureServer
```

Comment:
TypeScript typically does not use the `I*` naming convention, it could just be `ds.TemperatureServer`

Reply:
yeah, but then we have `TemperatureServer` and `ds.TemperatureServer`

Reply:
`ds.TemperatureServerSpec` ?

Reply:
Will change